scripts/pyparser/app.py

```diff
@@ -0,0 +1,148 @@
+from flask import Flask, jsonify, request
+import pyparsing as pp
+from pyparsing import oneOf
+import json
+import copy
+
+app = Flask(__name__)
+
+# https://utdallas.collegescheduler.com/api/terms/2021%20Spring/subjects/CS/courses/3345
+
+with open("data/new.json", 'r') as f:
+    data = json.loads(f.read())
+with open("data/major_requirements_processed.txt", 'r') as f:
+    degreePlan = json.loads(f.read())
+with open("data/ecs_computer-science.json", 'r') as f:
+    ecs_cs = json.loads(f.read())
+
+@app.route('/')
+def hello():
+    return "Hello World!"
+
+@app.route('/api/plan/ecs/computer-science')
+def degree():
+    return jsonify(ecs_cs)
+
+# @app.route('/degreeAll')
```

Comment:
Remove the commented code for this PR and stash it somewhere else if you're going to use it later.